doc/Interns/em-gray/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+# # May 7th, 2020
```

Comment:
Please add the time to this document...  "forgot" to mention" (don't worry, everybody gets the same note so you get a chance to deal with corrections and comments)

Reply:
Sounds good! Time has been added.